app/assets/stylesheets/bootstrap_and_overrides.scss

```diff
@@ -3,5 +3,144 @@ $brand-primary: #cc0000;
 // turn buttons square
 $border-radius-base: 0;
 
+
 @import "bootstrap-sprockets";
-@import "bootstrap"
+@import "bootstrap";
+
+body {
+  padding-bottom: 80px;
+}
+
+.navbar {
+  height: 60px;
+  background-color: #0E426C;
+  opacity: 0.80;
+  position: relative;
+  margin-bottom: 0px;
+
+  #logo img {
+    width: 100px;
+    margin-top: -27px;
+    }
+
+  .navbar-header{
+    height:59px;
+    }
+
+  ul.nav.navbar-nav{
+    margin-top: 0px;
+
+    .directory{
+      font-size: 2em;
+    }
+
+    li a{
+      height: 60px;
+      padding-top: 19px;
+    }
+
+    li a.dropdown-toggle.directory{
+      padding-top: 15px;
+    }
+
+    .divider{
+      margin:0;
+    }
+  }
+}
+
+.bottomMenu {
+  position: fixed;
+  bottom: 0;
+  width: 100%;
+  height: 60px;
+  border-top: 1px solid #000;
+  padding: 10px 0 0 30px;
+  background-color: #0E426C;
+  z-index: 99;
+  opacity: 0.80;
+
+  .navbar-left.footer {
+    .navbar-nav {
+      margin: 0;
+
+      li{
+        display: inline-block;
+
+        a{
+          padding-top:10px
+        }
+      }
+    }
+  }
+
+  .footer-text{
+    color:#D0D0D0;
+    margin: 10px 0 0 0;
+
+    a {
+      color:#FFF;
+    }
+  }
+
+  .footer-icons {
+    font-size: 2.1em;
+    margin: 0 35px;
+    float:right;
+
+    a {
+      color: #9d9d9d;
+    }
+
+    a:hover{
+      color:#FFF;
+      cursor: pointer;
+    }
+  }
+}
+
+@media (max-width: 767px) {
+  .navbar-fixed-top .navbar-collapse.collapse{
+    background-color: #0E426C;
+    max-height: none;
+    width: 330px;
+    float: right;
+    margin-right: -30px;
+    overflow: visible;
+
+    ul.nav li{
+      height: 40px;
+      a{
+        height: 40px;
+        padding-top: 10px;
+      }
+       a.directory, a.dropdown-toggle.directory{
```

Comment:
![image](https://cloud.githubusercontent.com/assets/5523172/15583625/7870e956-232b-11e6-820f-442d9ba92c7e.png)
☝️ `}` and `a.directory...` should start on the same column. See 👇 

```css
      }
      a.directory, a.dropdown-toggle.directory{
```